### PR TITLE
created test

### PR DESCRIPTION
**Pull request by Sofasmile**

### CODE REVIEW

File: src/main/java/core/basesyntax/Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Palindrome {
+
+    /**
+     * Реализуйте метод, проверяющий, является ли заданная строка палиндромом. Палиндромом называется
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Palindrome.java]: @Sofasmile please add <p> and </p> tags on each text section

Comment (boroda4436) [reply src/main/java/core/basesyntax/Palindrome.java]: p tags

Comment (Sofasmile) [reply src/main/java/core/basesyntax/Palindrome.java]: done

File: src/main/java/core/basesyntax/Palindrome.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+     *
+     * <p>Подсказки (не читайте, если хотите решить сами):
+     * для удаления из строки всех символов, не являющихся буквами и цифрами, можно воспользоваться
+     * регулярным выражением. Найдите в классе String метод, выполняющий замену по
+     * регулярному выражению;
+     * для перестановки символов строки в обратном порядке можно воспользоваться методом reverse(),
+     * который находится в классе StringBuilder;
+     * в классе String есть методы для преобразования всей строки в верхний и нижний регистр.
+     *
+     * <p>Пример: Madam, I'm Adam!
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Palindrome.java]:
```suggestion
     * <p>Пример: Madam, I'm Adam!</p>
```

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**